<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackCheckForm</class>
 <widget class="QWidget" name="BlackCheckForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="combo_Video">
         <item>
          <property name="text">
           <string>Camera Video0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera Video1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera Video2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>File Video</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Open">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_CLose">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Start">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Stop">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>添加监控区域</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="imageHLayout" stretch="10,5">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>365</width>
            <height>209</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>181</width>
            <height>209</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QFormLayout" name="formLayout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="statusBar">
     <property name="text">
      <string>状态信息</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Open</sender>
   <signal>clicked()</signal>
   <receiver>BlackCheckForm</receiver>
   <slot>on_button_OpenVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_CLose</sender>
   <signal>clicked()</signal>
   <receiver>BlackCheckForm</receiver>
   <slot>on_button_CloseVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Start</sender>
   <signal>clicked()</signal>
   <receiver>BlackCheckForm</receiver>
   <slot>on_button_StartVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Stop</sender>
   <signal>clicked()</signal>
   <receiver>BlackCheckForm</receiver>
   <slot>on_button_StopVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_Video</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>BlackCheckForm</receiver>
   <slot>on_combo_Video_Change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BlackCheckForm</receiver>
   <slot>on_button_AddRect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_OpenVideo_clicked()</slot>
  <slot>on_button_CloseVideo_clicked()</slot>
  <slot>on_button_StartVideo_clicked()</slot>
  <slot>on_button_StopVideo_clicked()</slot>
  <slot>on_combo_Video_Change(QString)</slot>
  <slot>on_button_AddRect_clicked()</slot>
  <slot>on_Label_LinkActivited(QString)</slot>
 </slots>
</ui>
